build_tools/docker/README.md

```diff
@@ -70,6 +70,29 @@ python3 build_tools/docker/manage_images.py --image all
 
 ## Adding or Updating an Image
 
+### Prerequisites
+
+## Instal GCloud
+
+To install gcloud run:
+
+```shell
+sudo apt install -y google-cloud-sdk
+```
+
+Follow the [`instructions in GCloud`][gcloud-help] or run:
+
+```shell
+gcloud auth login
+```
+
+By default Docker will require using `sudo`. If you are in this case make sure
+to authentify using:
+
+```shell
+sudo gcloud auth login
+```
+
```

Comment:
This isn't going to be the way to do it for everyone, and in particular I think it is only true for Googlers on glinux workstations. Probably no one else has this repository already configured. Let's instead link to the gcloud installation instructions (https://cloud.google.com/sdk/docs/install). They have a link at the top redirecting Googlers to the internal instructions. In general, we want to avoid rewriting other's help docs since then we're responsible for maintaining them.

``````suggestion
## Set Up GCloud

[Install the Google Cloud SDK](https://cloud.google.com/sdk/docs/install).

[Configure authentication for Docker](https://cloud.google.com/container-registry/docs/advanced-authentication#gcloud-helper)

By default Docker will require using `sudo`, so you need to
authenticate as the root user, e.g.:

```shell
sudo gcloud auth login
```

You can also
[configure Docker to not require sudo](https://docs.docker.com/engine/install/linux-postinstall/),
though bear in mind that this is equivalent to passwordless sudo
access.

``````